client/ctclient/ctclient.go

```diff
@@ -216,6 +219,63 @@ func getInclusionProofForHash(ctx context.Context, logClient *client.LogClient,
 	}
 }
 
+func getConsistencyProof(ctx context.Context, logClient *client.LogClient) {
+	if *treeSize == -1 {
```

Comment:
Does it make sense to allow size 0? I suppose it does no harm. Should it be < 0 then?

Reply:
Shifted both to check <= 0, to avoid edge cases.